modules/bmo-about-the-baremetalhost-resource.adoc

```diff
@@ -102,7 +102,7 @@ a|  (Optional) Contains the information about the RAID configuration for bare me
 ** `rotational`: If set to `true`, it will only select rotational disk drives. If set to `false`, it will only select solid-state and NVMe drives. If not set, it selects any drive types, which is the default behavior.
 ** `sizeGibibytes`: The size of the logical drive as an integer to create in GiB. If unspecified or set to `0`, it will use the maximum capacity of physical drive for the logical drive.
 
-* `softwareRAIDVolumes`: Contains the list of logical disks for software RAID. If you don't specify `rootDeviceHints`, the first volume is the root volume. If you set `HardwareRAIDVolumes`, this item will be invalid. Software RAIDs will always be deleted. The number of created software RAID devices must be `1` or `2`. If there is only one software RAID device, it must be `RAID-1`. If there are two RAID devices, the first device must be `RAID-1`, while the RAID level for the second device can be `0`, `1`, or `1+0`. The first RAID device will be the deployment device. Therefore, enforcing `RAID-1` reduces the risk of a non-booting node in case of a device failure. The `softwareRAIDVolume` field defines the desired configuration of the volume in the software RAID. The sub-fields are:
+* `softwareRAIDVolumes`: {product-title} {product-version} does not support software RAID. This configuration has the list of logical disks for software RAID. If you don't specify `rootDeviceHints`, the first volume is the root volume. If you set `HardwareRAIDVolumes`, this item will be invalid. Software RAIDs will always be deleted. The number of created software RAID devices must be `1` or `2`. If there is only one software RAID device, it must be `RAID-1`. If there are two RAID devices, the first device must be `RAID-1`, while the RAID level for the second device can be `0`, `1`, or `1+0`. The first RAID device will be the deployment device. Therefore, enforcing `RAID-1` reduces the risk of a non-booting node in case of a device failure. The `softwareRAIDVolume` field defines the desired configuration of the volume in the software RAID. The sub-fields are:
```

Comment:
Do we even need to keep the detailed description? We may simply redirect those curious to update docs (to highlight that it's not supported downstream).

Reply:
I did think about that. We're not allowed to xref in the docs to a github repo, so linking to here would not be allowed, if that's the upstream location. I could add a sentence for the softwareRAIDVolumes section, "OCP 4.x does not support software RAID. **The following information is for reference only.**" Would that help? 

Or if it's really of no use to the user, and doesn't appear in any default config files or anything, I could remove the softwareRAIDVolumes section.

Reply:
"The following information is for reference only" - like it!

Reply:
Thanks @dtantsur - I have made that change. Let me know if there's anything else or if you can give your LGTM